pages/common/docker-containers.md

```diff
@@ -0,0 +1,36 @@
+# docker container
+
+> Manage Docker containers.
+> More information: <https://docs.docker.com/engine/reference/commandline/container/>.
+
+- List currently running Docker containers:
+
+`docker container ls`
+
+- Start one or more stopped containers:
+
+`docker container start {{container_name}}`
```

Comment:
Perhaps we should add another container here to show that multiple containers are supported. That way we have a mix of single and multiple containers to give the idea that both are allowed.

Reply:
I have added that just in this one command (maybe it's enough).